uint32 tile_id
string obj_desire
uv cen_uv
uv f1th_uv
uv s2th_uv
uv t3th_uv
uv f4th_uv
